tests/python/integration/test_meta_schedule_auto_tensorize.py

```diff
@@ -326,7 +326,8 @@ def test_dp4a_conv2d():
     # )
 
 
-@pytest.mark.skip("Requires cascadelake")
+@tvm.testing.requires_vnni
+@pytest.mark.skip("Slow on CI")
```

Comment:
maybe just skip this in CI so it can still be run locally?
```suggestion
@pytest.mark.skip_if(tvm.testing.IS_IN_CI, reason="Slow on CI")
```